### PR TITLE
Update configuration to support configurable monitoring sinks.

### PR DESCRIPTION
**Pull request by vjkoskela**

### CODE REVIEW

File: conf/portal.application.conf [hunk 1]
```diff
@@ -91,14 +91,35 @@ portal.features {
...
   }
+
+  # Deprecated:
+  #uri = "http://localhost:7090"
+
+  # NOTES:
+  # - uri is deprecated
+  # - specifying uri disables sinks
+  # - for backwards compatibility uri takes precedence over sinks
+  # - migrate to sink as soon as possible
```
[the remark this answers is not in the document]
Comment (vjkoskela) [reply conf/portal.application.conf:122]: Fixed

File: pom.xml [hunk 2]
```diff
@@ -103,10 +103,10 @@
     <javax.inject.version>1</javax.inject.version>
     <jsr305.version>3.0.2</jsr305.version>
     <logback.steno.version>1.18.3</logback.steno.version>
-    <metrics.client.version>0.11.1</metrics.client.version>
-    <metrics.client.http.version>0.11.1</metrics.client.http.version>
-    <metrics.incubator.extra.version>0.11.0</metrics.incubator.extra.version>
-    <metrics.jvm.extra.version>0.11.0</metrics.jvm.extra.version>
+    <metrics.client.version>0.11.2</metrics.client.version>
```
[the remark this answers is not in the document]
Comment (vjkoskela) [reply pom.xml:106]: Fixed.

File: test/java/global/MainModuleTest.java [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+/**
+ * Tests for {@link MainModule}.
+
+ * @author Ville Koskela (ville dot koskela at inscopemetrics dot io)
+ */
+public final class MainModuleTest {
```
[the remark this answers is not in the document]
Comment (vjkoskela) [reply test/java/global/MainModuleTest.java:40]: Done!

File: pom.xml [hunk 4]
```diff
@@ -629,13 +629,13 @@
                 </volumes>
                 <wait>
                   <tcp>
-                    <mode>direct</mode>
+                    <mode>mapped</mode>
```
[the remark this answers is not in the document]
Comment (vjkoskela) [reply pom.xml:632]: Yes, I have having some local issues. Reverted.

**Review by cwbriones — APPROVED** [review]

**Pull request merged**